SparkPost/2web2ui
# Redux Upgrade - Attempting to upgrade Redux and related dependencies

[FE-1208](https://sparkpost.atlassian.net/browse/FE-1208)

### What Changed
- Upgrades the following Redux-related dependencies:
  + `redux`
  + `react-redux`
  + `redux-form`
  + `redux-thunk`
  + `raven-for-redux`
- Updates snapshots that reference `Connect(<Whatever>)` components
- Updates explicitly tests where use of Enzyme's `.find()` method were searching for `Connect(<Whatever>)`. Alternatively, the components were imported in to the tests. Ideally, this sort of testing strategy would be used less over time.

### How To Test
- Verify tests pass
- Go through core flows in the app and verify nothing is broken - probably worth doing with the console open

### To Do
